networkx/algorithms/approximation/tests/test_kcomponents.py

```diff
@@ -222,7 +222,7 @@ def setUp(self):
     def test_size(self):
         for G, A in self.GA:
             n = G.order()
-            s = len(G.edges())+len(A.edges())
+            s = len(list(G.edges())) + len(list(A.edges()))
```

Comment:
Instead of making a list I can do something like `sum(1 for _ in G.edges())`. What do you think?